app/src/androidTest/java/com/android/sample/ui/activityDetails/CommentItemTest.kt

```diff
@@ -75,7 +80,8 @@ class CommentItemTest {
           profileId = testProfileId,
           comment = testComment,
           onReplyComment = { _, _ -> },
```

Comment:
Adding a creatorId parameter to CommentItem and related test cases to identify the comment's creator.